lib/omnibus/stripper.rb

```diff
@@ -94,5 +94,99 @@ def strip_linux
         shellout!("chmod -x #{target}")
       end
     end
+
+    #
+    # Strip symbol from binaries on Windows. Note that this behavior differs from Linux.
+    #
+    # On Windows, DBG files are the original files, aka the un-stripped files.
+    # On Linux, DBG files are just symbol files.
+    #
+    # On Windows, pure symbol files can be used for non-live debugging e.g. dlv core c:\share\agent.dbg "C:\Share\agent.DMP"
+    # However, DLV cannot use the pure symbol files in live debugging:
+    #
+    #   C:\>dlv attach 6600 "C:\Program Files\Datadog\Datadog Agent\bin\agent.dbg"
+    #   could not attach to pid 6600: decoding dwarf section info at offset 0x0: too short
+    #
+    #   C:\Program Files\Datadog\Datadog Agent\bin>dlv attach 4024
+    #   could not attach to pid 4024: decoding dwarf section info at offset 0x0: too short
+    #
+    # To enable live debugging on Windows, we make DBG files on Windows the unstripped file.
+    # To perform live debugging, one must manually replace the stripped file with unstripped
+    # first, then can start debugging use DLV.
+    #
+    # To perform non-live debugging, the same command can be used, e.g. dlv core c:\share\agent.debug "C:\Share\agent.DMP"
+    #
+    def strip_windows
+      path = project.install_dir
+      log.debug(log_key) { "stripping on windows: #{path}" }
+
+      symboldir = File.join(path, ".debug")
+      log.debug(log_key) { "putting symbols here: #{symboldir}" }
+
+      if project.windows_files_to_strip && !project.windows_files_to_strip.empty?
+        project.windows_files_to_strip.each do |executable|
+          log.debug(log_key) { "processing: #{executable}" }
+          source = executable.strip
+          log.debug(log_key) { "processing source: #{source}" }
+
+          debugfile = "#{source}.debug"
```

Comment:
Maybe use `.dbg` for consistency with the Linux stripper (except if `.dbg` and/or `.debug` extensions have specific meanings on Windows)?

Reply:
".dbg" was an extension name for symbol files on Windows and DOS (now it's ".sym").